src/mmap.rs

```diff
@@ -507,18 +410,37 @@ mod tests {
     }
 
     #[test]
+    #[cfg(unix)]
     fn map_invalid_size() {
         let e = MmapRegion::new(0).unwrap_err();
         assert_eq!(e.raw_os_error(), Some(libc::EINVAL));
     }
 
     #[test]
+    #[cfg(windows)]
```

Comment:
Could we move these test cases into mmap_unix and mmap_windows? So we could remove many condition compilations.